doc/faq/index.rst

```diff
@@ -0,0 +1,82 @@
+.. _faq:
+
+==========================
+Frequently asked questions
+==========================
+
+.. _faq_general:
+
+General
+=======
+
+What happened to import dicom?
+------------------------------
+Starting in version 1.0, *pydicom* changed the library import from
+``import dicom`` to ``import pydicom``. If you're used to using the earlier
+versions of *pydicom* see the :doc:`transitioning guide
+<../old/transition_to_pydicom1>` on how to make the change.
+
+How do I cite pydicom?
+----------------------
+
+The easiest method is probably to `find the Zenodo DOI
+<https://zenodo.org/search?page=1&size=20&q=conceptrecid:1291985&all_versions&sort=-version>`_
+for the version you are using and then entering your required citation style
+in the *Cite as* box.
+
+Alternatively, you can use something along the lines of:
+
+.. code-block:: text
+
+  Mason, D. L., et al, pydicom: An open source DICOM library, https://github.com/pydicom/pydicom [Online; accessed YYYY-MM-DD].
+
+
+.. _faq_install:
+
+Installation
+============
+
+What are pydicom's prerequisites?
+---------------------------------
+
+Required
+~~~~~~~~
+*pydicom* requires Python.
+
```

Comment:
I'm good with `*pydicom*` in general, but I would suggest capitalizing at the start of sentences.  I used to always write it lower case, but I personally thought it looked strange on reading it later in places where words would normally be capitalized.

Reply:
I find it awkward both ways but prefer it to be consistent (i.e. always Pydicom or always pydicom).

[Wikipedia style guide](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Capital_letters#Initial_letters_in_sentences_and_list_items) says to keep it uncapitalised, like eBay and iPad.

Reply:
Yes - I have looked around at other libraries, and they all seem to use that (e.g. non-capitalized names at sentence start). So I guess we may leave it at that... Though @darcymason shall have the last word, of course :)